app/assets/javascripts/config.js.coffee.erb

```diff
@@ -13,15 +13,21 @@
   match = /cmr_env=(\w+)/.exec(query)
   cmr_env = if match then match[1] else '<%= Rails.configuration.cmr_env %>'
 
+
```

Comment:
Add this to line#16 
```reverb_url = "<%=ENV['reverb_root'] || Rails.configuration.services['edsc'][Rails.env]['reverb_root']%>"```
and remove line#18, 23, 27, 28, and 30, since ```cmr_env``` changes based on query (see line#12 and 13) and we don't want it to point to a different reverb if cmr_env is different than rails env.